README.md

```diff
@@ -103,6 +111,9 @@ The provided data-readers' tags are:
 - `#config/env` Tags a string, interpreted as the name of an environment variable, and yields the string value of the environment variable. If the environment does not have that entry, then the var will use its default value or remain unbound.
 - `#config/file` Tags a string, interpreted as a path to a file, and yields the string contents of the file. If the file does not exist, then the var will use its default value or remain unbound.
 - `#config/edn` Tags a string, interpreted as a single EDN-formatted object, and yields the read object.  When composed with `#config/env` or `#config/file`, if the external value is not provided, then the var will use its default value or remain unbound.
+- `#config/etcd` Tags a string, interpreted as a key to lookup a value in etcd.
+- `#config/lookup` Tags a string, looks up a config value from another key in
```

Comment:
@pjstadig: The use case for #config/lookup is to solve problems like having multiple db configurations across projects... an example:

```clojure
 outpace.targeting-engine.offers-db.cli/db-config
 {:user "foo",
  :password "password",
  :server "my-db",
  :port 1434,
  :dbname "foo-db"}

 outpace.targeting-engine.offers-db.cli/test-db-config
 {:user "foo",
  :password "password",
  :server "my-db",
  :port 1434,
  :dbname "foo-db"}
```

```clojure
 the-db
 {:user "foo",
  :password "password",
  :server "my-db",
  :port 1434,
  :dbname "foo-db"}

 outpace.targeting-engine.offers-db.cli/db-config
 #config/lookup "the-db"

 outpace.targeting-engine.offers-db.cli/test-db-config
 #config/lookup "the-db"
```